lib/features/mailbox_dashboard/presentation/mixin/user_setting_popup_menu_mixin.dart

```diff
@@ -57,17 +57,33 @@ mixin UserSettingPopupMenuMixin {
     return const SizedBox.shrink();
   }
 
-  Widget _settingAction(BuildContext context, Function? onCallBack) {
+  Widget _settingAction(BuildContext context, VoidCallback? onCallBack) {
     return PopupMenuItemWidget(
-        _imagePaths.icSetting,
-        AppLocalizations.of(context).manage_account,
-        () => onCallBack?.call());
+      name: AppLocalizations.of(context).manage_account,
+      icon: _imagePaths.icSetting,
+      padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 16),
+      textStyle: Theme.of(context).textTheme.labelMedium?.copyWith(
+        fontSize: 15,
+        color: Colors.black,
+        fontWeight: FontWeight.w500
+      ),
+      space: 12,
```

Comment:
Found it also appear inside `_logoutAction`  widget. How about reuse?

Reply:
Good detection.

Reply:
Done